sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -147,10 +147,10 @@ class DataSourceV2SQLSuite
       Array("", "", ""),
       Array("# Detailed Table Information", "", ""),
       Array("Name", "testcat.table_name", ""),
-      Array("Comment", "this is a test table", ""),
-      Array("Location", "file:/tmp/testcat/table_name", ""),
-      Array("Provider", "foo", ""),
       Array(TableCatalog.PROP_OWNER.capitalize, defaultUser, ""),
+      Array("Provider", "foo", ""),
+      Array("Location", "file:/tmp/testcat/table_name", ""),
+      Array("Comment", "this is a test table", ""),
```

Comment:
It would be great if we don't change this order in this **refactoring** if possible. 
Do you think we can avoid this, @AngersZhuuuu ?

Reply:
> It would be great if we don't change this order in this **refactoring** if possible. Do you think we can avoid this, @AngersZhuuuu ?

Code will be complex, how about use `sort` to make it have a stable order?

Reply:
Are you saying that the order of output is different after your change?

Reply:
> 

Hmmm, dose the order matter? We can add an sort like normal properties to make it stable.

Reply:
@AngersZhuuuu . Yes, unfortunately, it matters. Personally, I'm -1 if there is no way to avoid the side-effects.
> Hmmm, dose the order matter? We can add an sort like normal properties to make it stable.